addons/pos_self_order/static/src/mobile/self_order_mobile_service.js

```diff
@@ -36,6 +36,16 @@ export class SelfOrder extends selfOrderCommon {
                 [this]
             );
         }
+        this.bus_service.addChannel(`pos_config-${this.access_token}`);
+        this.bus_service.addEventListener("notification", ({ detail }) => {
+            for (const message of detail) {
+                if (message.type === "PRODUCT_CHANGED") {
+                    this.handleProductChanges(message.payload);
+                } else {
+                    this.dispatchMessage(message);
```

Comment:
This function do not exist in mobile. But anyway wait the merging of self & kiosk